R/model_run.R

```diff
@@ -1,4 +1,5 @@
 model_run <- function(parameters, model_version) {
   # Returning sample response for now
-  to_json(read_local_json("sample_scenario_results_response.json"), auto_unbox = TRUE)
+  from_file <- read_local_json("sample_scenario_results_response.json")
+  jsonlite::toJSON(from_file, auto_unbox = TRUE)
```

Comment:
For reasons which I cannot explain, after the util rename, `model_run` wasn't able to find it anymore in the e2e test.... I've no idea why! But since this is going to be replaced with a call to the daedalus package soon, maybe we can just not use the util here (for now).